src/home_content.json

```diff
@@ -0,0 +1,144 @@
+{ 
+	"sections" : [
+		{ 
+			"title": "Getting Started" ,
+			"content" :  [
+			    { 
+					"par": "The AnitaB.org community is a fusion of rich values and diverse cultures consisting of members all across the seven oceans with a fearless mission of inspiring ,motivating and helping women with their involvement in technical fields and has made a significant impact with regards to contribution of women in tech.\n The other values of the organization include Accountable,Agile,Curious,Courageous,&Diversity,Equity,Inclusion (DEI),encourage everyone to follow the org’s contributing guidelines and obviously passion for open source. "
+
+				},
+				{
+					"par": "Zulip is our main communication platform and you should join us there and first give a short introduction about yourself in the #newcomers stream.Zulip consists of various streams like Documentation,Coding,Design,etc and the one that fascinates you the most is the one you should start contributing in.\n The #announcements stream will be displaying all the important announcements.Tag our community members if there's any confusion or ask for proper guidance from them regarding your contributions to the community."
+				},
+				{  
+					"par": "Our code of conduct is simple- 'Be nice,Be respectful' which is the one of the many reasons the community coexists in harmony .The members should contribute actively keeping the policies of being repectful to everyone and refrainging from conflicts, unruly behaviour and speech.Anyone who wishes to join the community is expected to follow these guidelines.\n If you are subject to or witness unacceptable behavior, or have any other concerns,please notify a community organizer or adminas soon as possible."
```

Comment:
```suggestion
					"par": "Our code of conduct is simple- 'Be nice,Be respectful' which is the one of the many reasons the community coexists in harmony .The members should contribute actively keeping the policies of being respectful to everyone and refraining from conflicts, unruly behaviour and speech. Anyone who wishes to join the community is expected to follow these guidelines.\n If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer or admins as soon as possible."
```

Reply:
Thanks @isabelcosta .I'll make the changes soon.Just a question , I didnt get what changes have you requested here.Could you please explain .Thanks in advance:)

Reply:
> this one@annabauza

Reply:
this one @annabauza